build.sh

```diff
@@ -1134,8 +1139,11 @@ EOM
     "package-ios-swift")
         cd tightdb_objc
         for version in 8.0 8.1 8.2; do
-            REALM_XCODE_VERSION="$version" sh build.sh prelaunch-simulator
-            REALM_XCODE_VERSION="$version" sh build.sh ios-swift
+            REALM_XCODE_VERSION=$version
+            REALM_SWIFT_VERSION=
+            set_xcode_and_swift_versions
```

Comment:
Both are actually needed, since `set_xcode_and_swift_versions` is the only one `export`ing it (so it can go to a child process), and it is also always setting `REALM_SWIFT_VERSION` so without that being inside the loop you would get it set to the value for the first one.